packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -112,11 +112,20 @@ module.exports = class SingleSpaReactGenerator extends Generator {
     this.fs.extendJSON(this.destinationPath("package.json"), packageJson);
 
     if (this.options.typescript) {
+      const typescriptPackageJsonTemplate = await fs.readFile(
+        this.templatePath("../../common-templates/typescript/package.json.ejs"),
+        "utf-8"
+      );
+      const typescriptPackageJsonStr = ejs.render(
```

Comment:
Would it be simpler to have a separate `reactTypes.json` file instead? My thought process is that this isn't a true package.json (at least not one that contains everything necessary for an npm project), and it would avoid having to ejs render them. I'm failing to think of any benefits that I could be overlooking though.

Reply:
sure

Reply:
I have updated this with your feedback